srcpkgs/YACReader/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'YACReader'
+pkgname=YACReader
+version=9.5.0
+revision=1
+wrksrc=yacreader-${version}
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-script-devel qt5-declarative-devel
+ qt5-quickcontrols qt5-multimedia-devel qt5-imageformats poppler-qt5-devel
+ qrencode-devel glu-devel libunarr-devel"
+makedepends="qt5-script-devel qt5-declarative-devel qt5-quickcontrols
+ qt5-multimedia-devel qt5-imageformats poppler-qt5-devel qrencode-devel
+ glu-devel libunarr-devel"
+depends="qt5-imageformats qrencode"
```

Comment:
Is the `qrencode` binary actually used? Looks to me like it only uses the library which gets pulled in automatically.
There is also no sign of `qt5-imageformats` being used.

Reply:
They’re optional, so I left them in just in case, but they probably aren’t used at runtime. Whether `qt5-imageformats` is used at all, I do not know, but it’s listed as a dependency.

Reply:
The dependency list seems to be outdated / incorrect, it doesn't link against `qt5-imageformats`